java/src/test/java/com/saucelabs/simplesauce/EdgeTest.java

```diff
@@ -68,7 +65,8 @@ public void withEdge14_returnsBrowserVersion14_14393() {
     @Test
     public void withEdge13_returnsBrowserVersion13_10586() {
         sauceOptions.withEdge13();
-        sauce = instantiateSauceSession();
+        startSauceSession();
+        Mockito.doReturn(dummyRemoteDriver).when(sauce).createRemoteWebDriver();
```

Comment:
`Mockito.doReturn(dummyRemoteDriver).when(sauce).createRemoteWebDriver();`
and `sauce.start();` are probably not required, right?

Reply:
Yeah, until we separate out the options, things aren't set until we call `start()` That's next to fix. :)

Reply:
oh, I see what you are saying, yes, don't need both.